discretisedfield/plotting/pyvista_mesh.py

```diff
@@ -0,0 +1,160 @@
+import pyvista as pv
+import ubermagutil.units as uu
+
+import discretisedfield.plotting.util as plot_util
+
+
+class PyVistaMesh:
+    def __init__(self, mesh):
+        if mesh.region.ndim != 3:
+            raise RuntimeError("Only 3d meshes can be plotted.")
+        self.mesh = mesh
+
+    def __call__(
+        self,
+        *,
+        plotter=None,
+        color=plot_util.cp_hex,
```

Comment:
The mutable default is potentially dangerous. Not sure if we would want to do anything.

Reply:
Agreed - this is what we did for `k3d`. I would guess it's okay for now as we have never heard of any issues.